cmd/plural/deploy.go

```diff
@@ -292,7 +294,7 @@ func handleDiff(c *cli.Context) error {
 	fmt.Printf("Diffing applications [%s] in topological order\n\n", strings.Join(sorted, ", "))
```

Comment:
This is one of the debug outputs that should be removed. Very strange that it isn't in the git diff though.

Reply:
Nevermind, I'm mistaken.